File: workspace-servidor/flotix2021/src/main/java/com/flotix/response/bean/ServerResponseCliente.java
package com.flotix.response.bean;

import java.util.List;

import com.flotix.dto.ClienteDTO;

public class ServerResponseCliente {

	private String idCliente = null;
	private List<ClienteDTO> listaCliente = null;

	private ErrorBean error = new ErrorBean();

	public ServerResponseCliente() {
		super();
	}

	public String getIdCliente() {
		return idCliente;
	}

	public void setIdCliente(String idCliente) {
		this.idCliente = idCliente;
	}

	public List<ClienteDTO> getListaCliente() {
		return listaCliente;
	}

	public void setListaCliente(List<ClienteDTO> listaCliente) {
		this.listaCliente = listaCliente;
	}

	public ErrorBean getError() {
		return error;
	}

	public void setError(ErrorBean error) {
		this.error = error;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelResponse/ServerResponseCliente.cs
using Flotix2021.ModelDTO;
using System.Collections.Generic;

namespace Flotix2021.ModelResponse
{
    class ServerResponseCliente
    {
        private string _idCliente;

        public string idCliente
        {
            get { return _idCliente; }
            set { _idCliente = value; }
        }

        private List<ClienteDTO> _listaCliente;

        public List<ClienteDTO> listaCliente
        {
            get { return _listaCliente; }
            set { _listaCliente = value; }
        }

        private ErrorBean _error;

        public ErrorBean error
        {
            get { return _error; }
            set { _error = value; }
        }
    }
}
